### PR TITLE
feat(modal, wizard): remove shim mapping and fallbacks

### PR DESCRIPTION
**Pull request by valentin-mladenov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1266, CDE-1260

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/modal/_modal.clarity.scss [hunk 1]
```diff
@@ -126,70 +74,29 @@
...
-        line-height,
-        clr-modal-title-line-height,
-        modal-variables.$clr-modal-title-line-height,
-        variables.$clr-use-custom-properties
-      );
+      font-family: modal-variables.$clr-modal-title-font-family;
+      font-size: modal-variables.$clr-modal-title-font-size;
+      font-weight: modal-variables.$clr-modal-title-font-weight;
+      line-height: modal-variables.$clr-modal-title-line-height;
       letter-spacing: modal-variables.$clr-modal-title-letter-spacing;
```
Comment (dtsanevmw) [inline projects/angular/src/modal/_modal.clarity.scss:87]: Do we have a correct typography from the mixin?

File: projects/angular/src/modal/_modal.clarity.scss [hunk 2]
```diff
@@ -256,57 +148,18 @@
...
     }
-
-    .modal-footer {
-      @include mixins.css-var(
-        padding,
-        clr-modal-sm-footer-padding,
-        variables.$clr_baselineRem_0_5 variables.$clr_baselineRem_1 0 0,
-        variables.$clr-use-custom-properties
-      );
-    }
```
Comment (dtsanevmw) [inline projects/angular/src/modal/_modal.clarity.scss:288]: Are you sure we need to remove all those?

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 3]
```diff
@@ -25,129 +27,52 @@
...
-        clr-wizard-main-text-color,
-        wizard-variables.$clr-wizard-main-text-color,
-        variables.$clr-use-custom-properties
-      );
+      color: wizard-variables.$clr-wizard-main-text-color;
       width: 100%;
     }
 
-    .modal-footer {
+    .modal-footer.clr-wizard-footer {
```
Comment (dtsanevmw) [inline projects/angular/src/wizard/_wizard.clarity.scss:68]: This is already under `clr-wizard` so except for more specificity is that needed?

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 4]
```diff
@@ -25,129 +27,52 @@
...
-        clr-wizard-footer-height,
-        wizard-variables.$clr-wizard-footer-height,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        padding,
-        clr-wizard-footer-padding,
-        wizard-variables.$clr-wizard-default-space wizard-variables.$clr-wizard-default-space 0 0,
-        variables.$clr-use-custom-properties
-      );
```
Comment (dtsanevmw) [inline projects/angular/src/wizard/_wizard.clarity.scss:150]: I see it was `  --clr-wizard-footer-padding: 0 var(--clr-wizard-main-padding) var(--clr-wizard-main-padding) var(--clr-wizard-main-padding);` in the shim, is this reworked in some way to be the same?

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 5]
```diff
@@ -291,116 +166,34 @@
...
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          background-color,
-          clr-wizard-stepnav-link-hover-bg-color,
-          transparent,
-          variables.$clr-use-custom-properties
-        );
+        color: wizard-variables.$clr-wizard-stepnav-text--active;
+        background-color: transparent;
```
Comment (dtsanevmw) [inline projects/angular/src/wizard/_wizard.clarity.scss:178]: I see it was `--clr-wizard-stepnav-link-hover-bg-color: #{tokens.$cds-alias-object-interaction-background-hover};` but now transparent?

File: projects/angular/src/wizard/_wizard.clarity.scss [hunk 6]
```diff
@@ -558,13 +285,7 @@
...
     .clr-wizard-page:not([aria-hidden='true']) {
       display: block;
-      @include mixins.css-var(
-        padding,
-        clr-wizard-page-padding,
-        variables.$clr_baselineRem_0_75 wizard-variables.$clr-wizard-default-space
-          wizard-variables.$clr-wizard-default-space wizard-variables.$clr-wizard-default-space,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
```
Comment (dtsanevmw) [inline projects/angular/src/wizard/_wizard.clarity.scss:288]: I think we are not adding padding anywhere so maybe that's not needed.

**Review by dtsanevmw — CHANGES_REQUESTED** [review]
Overall looks good, I just left some questions worth checking.

File: projects/angular/src/modal/_modal.clarity.scss (hunk 1 above)
Comment (valentin-mladenov) [reply projects/angular/src/modal/_modal.clarity.scss:87]: We have them and are used in [line 78](https://github.com/vmware-clarity/ng-clarity/blob/shim-removal-wizard_cde-1266/projects/angular/src/modal/_modal.clarity.scss#L78). This rule is to override it when title is `not([cds-text])`.

File: projects/angular/src/wizard/_wizard.clarity.scss (hunk 6 above)
Comment (valentin-mladenov) [reply projects/angular/src/wizard/_wizard.clarity.scss]: removed in commit b19c78a

File: projects/angular/src/wizard/_wizard.clarity.scss (hunk 5 above)
Comment (valentin-mladenov) [reply projects/angular/src/wizard/_wizard.clarity.scss]: Corrected in commit b19c78a

File: projects/angular/src/wizard/_wizard.clarity.scss (hunk 3 above)
Comment (valentin-mladenov) [reply projects/angular/src/wizard/_wizard.clarity.scss]: removed in commit b19c78a

**Review by dtsanevmw — APPROVED** [review]

**Pull request merged**